apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -64,113 +81,87 @@ export default function Rundown({ data }: RundownProps) {
-            reorderEvent(cursor, nextIndex - 1, nextIndex);
-          }
-          // Alt + Ctrl + Arrow Up
-        } else if (event.code == 'ArrowUp') {
-          const { previousEvent, previousIndex } = getPreviousNormal(rundown, order, cursor);
-          if (previousEvent && previousIndex !== null) {
-            reorderEvent(cursor, previousIndex + 1, previousIndex);
-          }
-        }
+  const selectEntry = useCallback(
+    (cursor: string | null, direction: 'up' | 'down') => {
+      if (order.length < 1) {
+        return;
+      }
+      const newCursor =
+        direction === 'up'
+          ? cursor === null
+            ? getLastNormal(rundown, order)?.id //<-- nothing selected and arrow down gives the first entry
+            : getPreviousNormal(rundown, order, cursor ?? '').entry?.id
+          : cursor === null
+          ? getFirstNormal(rundown, order)?.id //<-- nothing selected and arrow up gives the last entry
+          : getNextNormal(rundown, order, cursor ?? '')?.entry?.id;
+
+      if (newCursor) {
+        setCursor(newCursor);
       }
     },
-    [order, cursor, rundown, setCursor, insertAtCursor, reorderEvent],
+    [order, rundown, setCursor],
   );
 
+  const moveEntry = useCallback(
+    (cursor: string | null, direction: 'up' | 'down') => {
+      if (order.length < 2 || cursor == null) {
+        return;
+      }
+      const { index } =
+        direction === 'up' ? getPreviousNormal(rundown, order, cursor) : getNextNormal(rundown, order, cursor);
+
+      if (index !== null) {
+        const offsetIndex = direction === 'up' ? index + 1 : index - 1;
+        reorderEvent(cursor, offsetIndex, index);
+      }
+    },
+    [order, reorderEvent, rundown],
+  );
+
+  // shortcuts
+  useHotkeys([
+    ['alt + ArrowDown', () => selectEntry(cursor, 'down'), { preventDefault: true }],
+    ['alt + ArrowUp', () => selectEntry(cursor, 'up'), { preventDefault: true }],
+    ['alt + ctrl + ArrowDown', () => moveEntry(cursor, 'down'), { preventDefault: true }],
+    ['alt + ctrl + ArrowUp', () => moveEntry(cursor, 'up'), { preventDefault: true }],
+
+    ['Escape', () => setCursor(null), { preventDefault: true }],
+
+    ['ctrl + Backspace', () => deleteAtCursor(cursor), { preventDefault: true }],
+
+    ['alt + E', () => insertAtCursor(SupportedEvent.Event, cursor), { preventDefault: true }],
+    ['alt + shift + E', () => insertAtCursor(SupportedEvent.Event, cursor, true), { preventDefault: true }],
+
+    ['alt + B', () => insertAtCursor(SupportedEvent.Block, cursor), { preventDefault: true }],
+    ['alt + shift + B', () => insertAtCursor(SupportedEvent.Block, cursor, true), { preventDefault: true }],
+
+    ['alt + D', () => insertAtCursor(SupportedEvent.Delay, cursor), { preventDefault: true }],
+    ['alt + shift + D', () => insertAtCursor(SupportedEvent.Delay, cursor, true), { preventDefault: true }],
+
+    ['ctrl + C', () => setEntryCopyId(cursor), { preventDefault: true }],
```

Comment:
this `ctrl` seems to always refer to ctrl and not command in mac, this may be confusing
I also wonder if we are getting into conflicts with the browser shortcuts, it would be good to test

Reply:
hmm yeah, the relevant key on mac is Meta right?
`ctrlOrCmd = isMacOS() ? 'meta' : 'ctrl';`